keras_eval/eval.py

```diff
@@ -145,6 +149,32 @@ def evaluate(self, data_dir=None, top_k=1, filter_indices=None, confusion_matrix
 
         return self.probabilities, self.labels
 
+    def evaluate_combination_concepts(self):
+        '''
+        Sets probabilities if model concepts are to be combined in test
```

Comment:
Can we rephrase it to include what the function uses (`self.concept_dictionary`, `self.probabilities`)?

Something like:
> Combine probabilities following `self.concept_dictionary` criteria and rewrite them in `self.probabilities`